blocks/header/header.css

```diff
@@ -28,41 +28,169 @@ body {
     display: block;
 }
 .header__title {
-margin-top:4px;
+    margin-top: 4px;
 }
 .header__subtitle {
     font-family: 'Roboto Slab','Arial', sans-serif;
     font-size: 100px;
     font-weight: 400;
 
     margin-top: 16px;
-    margin-bottom:0;
+    margin-bottom: 0;
 
     text-align: center;
     letter-spacing: -3px;
-    text-transform:uppercase;
+    text-transform: uppercase;
 
     color: #0d3351;
 }
 .header__button {
-    display: block;
-    margin-top: 33px;
-    margin-left: auto;
-    margin-right: auto;
+    display: flex;
+    height: 33px;
+    margin: 29px auto;
+    align-items: center;
+    justify-content: center;
 }
 .decor-line {
-    z-index: 2;
-
     margin-right: auto;
-    margin-left:auto;
+    margin-left: auto;
     position: absolute;
 
     left: 0;
     right: 10px;
-    top:470px;
+    top: 470px;
 }
 .header__info {
     display: flex;
     align-items: center;
-    margin-top:31px;
+    margin-top: 31px;
+}
+@media (min-width: 768px) and (max-width: 1300px) {
+    body {
+        max-width: 1300px;
+    }
+    .header {
+        height: 600px;
+
+        background-image: url(../../assets/img/header/header_tablet.png);
+    }
+    .header__container {
+        max-width: 722px;
+    }
+    .header__logo {
+        width: 70px;
+        height: 70px;
+        
+        margin-top: 78px;
+    }
+    .header__title {
+        letter-spacing: 1.5px;
+
+        margin-top: 0;     
+    }
+    .header__subtitle {
+        font-size: 72px;
+        letter-spacing: -2px;
+
+        margin-top: -15px;
+    }
+    .header__button {
+        margin-top: 5px;
+    }
+    .decor-line {
+        width: 45px;
+        height: 222px;
+
+        right: 20px;
+        top: 415px;
+    } 
+    .header__info {
+        margin-top: 26px;
+    }
+}
+@media (min-width:321px) and (max-width:767px) {
+    body{

```

Comment:
I think you don't need this style because the website is full-screen until 1300.